NEWS.md

```diff
@@ -226,6 +226,10 @@ Language changes
   * `try` blocks without `catch` or `finally` are no longer allowed. An explicit empty
     `catch` block should be written instead ([#27554]).
 
+  * `AbstractArray` types that use unconventional (not 1-based) indexing can now support
+    `size`, `length`, and `@inbounds`. To optionally enforce conventional indices,
+    you can `@assert is_one_indexed(A)`.
```

Comment:
This still has the old name here.  I find the `!is_non1_indexed` suboptimal.  What about `is_offset_indexed` or `has_offset_axes`?

Reply:
Oooh, I like those! Will change to `has_offset_axes`.